packages/cashier/src/_common/utility.js

```diff
@@ -1,13 +1,4 @@
 import { getCurrencyDisplayCode } from '@deriv/shared';
-
-const template = (string, content) => {
```

Comment:
@jim-deriv We have another duplicate one in `packages/core/src/_common/utility.js`
Can you check it too?

Reply:
I'll be removing this template here as well

Reply:
I have removed the ones you've suggested though.